Cargo.toml

```diff
@@ -41,7 +41,7 @@ async-trait = "0.1.41"
 femme = { version = "2.1.1", optional = true }
 futures-util = "0.3.6"
 http-client = { version = "6.1.0", default-features = false }
-http-types = "2.5.0"
+http-types = { version = "2.10.0", default-features = false }
```

Comment:
This also disables the `async-std` feature, which we rely on for `Body::from_file`. We should probably re-enable this feature.

Reply:
Strange! I did test this with `cargo test --no-default-features --features h1-server,logger`, which build-tested examples that call `Body::from_file`. It looks like something *else* is pulling in the `http-types` "default" feature.

I'll fix `tide` to require the `fs` feature of `http-types`, and also look into what else in the dependency tree is still pulling in `http-types` with default features enabled.